### PR TITLE
feat: landing page and time display

### PR DESCRIPTION
**Pull request by krgarima**
##Change Logs:

#Description: Added Landing page and time

## More details:
- Added `Manifest.json`
- `Wallpaper` changes on each refresh
- Added a welcome user page where user will enter his/her `name`.
- On clicking the continue button, user will be will shown `current time`, a `salutation` with his name
- All data (entered name) is stored in the `local storage` so data is not lost even if the user closes the tab

Screenshots:
ext1
ext2

### CODE REVIEW

**Review by Asuraking1n — APPROVED** [review]
UI looks awesome....
Code part was also cool...

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+}
+
+.userNameInput {
+  visibility: visible;
+}
+
+.display {
+  margin: 3rem;
+  display: none;
+  background-color: rgba(0, 0, 0, 0.316);
```
Comment (athar05) [inline styles.css:53]: You can consider making this a variable, as you have used it a couple of times

**Review by athar05 — COMMENTED** [review]
Overall code looks clean.

**Comment by athar05** [comment]
The UI good. I liked the implementation of the greeting to the user with the switch statement. I think you can merge it.

**Pull request merged**